etc/nginx/nginx.conf.ctmpl

```diff
@@ -61,4 +61,4 @@ http {
         }
         {{end}}
     }
-}
+}
```

Comment:
As with all such things, some people are pedantic in the opposite way.

The Github web editor removes trailing newlines, but I don't believe that's at issue here.